components/raftstore/src/store/fsm/apply.rs

```diff
@@ -2111,6 +2118,8 @@ where
 
         let state = if self.pending_remove {
             PeerState::Tombstone
+        } else if self.wait_data {
+            PeerState::Unavailable
```

Comment:
Does it reset to `Normal` after applying the snapshot?

Reply:
Yes, the apply snap worker will change this state.